tests/common/devices.py

```diff
@@ -337,7 +337,7 @@ def critical_process_status(self, service):
         # get critical process list for the service
         output = self.command("docker exec {} bash -c '[ -f /etc/supervisor/critical_processes ] && cat /etc/supervisor/critical_processes'".format(service), module_ignore_errors=True)
         for l in output['stdout'].split():
-            critical_process_list.append(l.rstrip())
+            critical_process_list.append(l.split(':')[1].rstrip())
```

Comment:
can you check if ':' exists? if yes, then do the split, if not, then do not split. I do not know if this criticial_processes_list has been ported back to 201911 branch.

Reply:
> can you check if ':' exists? if yes, then do the split, if not, then do not split. I do not know if this criticial_processes_list has been ported back to 201911 branch.

Yes. I checked containers in  SONiC.20191130.40, and the ':' exists. One of the output is below:
```
root@str-msn2700-02:/# cat /etc/supervisor/critical_processes
program:orchagent
program:portsyncd
program:neighsyncd
program:vlanmgrd
program:intfmgrd
program:portmgrd
program:buffermgrd
program:vrfmgrd
program:nbrmgrd
program:vxlanmgrd
```

Reply:
sorry, I meant the test to check if there is colon, if yes, then split, if no, the no split. this is the improve the test backward compatibility.

Reply:
> sorry, I meant the test to check if there is colon, if yes, then split, if no, the no split. this is to improve the test backward compatibility.

Updated. Thanks!